### ISSUE TITLE
links

### ISSUE THREAD

**Issue by daglaroglou** [ClaqNode-Hosting/stellar-theme#2]
all links are invalid... whats wrong?

**Comment by ClaqNode-Hosting** [comment]
What link is invalid?

**Comment by daglaroglou** [comment]
the discord invite, the demo view, the zip file... all

**Comment by ClaqNode-Hosting** [comment]
Owh, The zip stellar the link is expired, I'll update it....  Later...

**Comment by daglaroglou** [comment]
any new invite link btw?

**Comment by ClaqNode-Hosting** [comment]
invite link what? Discord server?? I want an update now zip the stellar file.

**Comment by daglaroglou** [comment]
theme doesnt work

**Comment by ClaqNode-Hosting** [comment]
The script just updated so, maybe vps you can not to see the script that I updated earlier.

**Comment by ClaqNode-Hosting** [comment]
Try you download theme-installer.se in your vps and rename the file so install.sh and bash install.sh

**Comment by daglaroglou** [comment]
still doesnt work, now i get display error on server and now its white screen

**Comment by ClaqNode-Hosting** [comment]
Make sure your panel is still new and never install any theme, it can cause your old theme to stay.

**Comment by daglaroglou** [comment]
yeah i just finished setting up panel and node, all work fine, never added even a single theme or addon
